wolfcrypt/src/wc_port.c

```diff
@@ -3164,6 +3164,30 @@ time_t z_time(time_t * timer)
 {
     struct timespec ts;
 
+    #if defined(CONFIG_RTC) && \
+        (defined(CONFIG_PICOLIBC) || defined(CONFIG_NEWLIB_LIBC))
+    /* Try to obtain the actual time from an RTC */
+    static const struct device *rtc = DEVICE_DT_GET(DT_NODELABEL(rtc));
+
+    if (device_is_ready(rtc)) {
+        struct rtc_time rtc_time;
+        struct tm *tm_time = rtc_time_to_tm(&rtc_time);
+
+        int ret = rtc_get_time(rtc, &rtc_time);
+
+        if (ret == 0) {
+            time_t epochTime = mktime(tm_time);
+
+            if (timer != NULL)
+                *timer = epochTime;
+
+            return epochTime;
+        }
+    }
+    #endif
+
+	/* Fallback to uptime since boot. This works for relative times, but
```

Comment:
```8 bit, trailing spaces or tabs:
./wolfcrypt/src/wc_port.c:3189:»/* Fallback to uptime since boot. This works for relative times, but
./wolfcrypt/src/wc_port.c:3190:» * not for ASN.1 date validation */
weird control chars, hard tabs, CRs, trailing whitespace:
```

Reply:
fixed